### PR TITLE
SEODescriptionAdditions: User Story 1657424, part4

### PR DESCRIPTION
**Pull request by damabe**
## Summary

https://dev.azure.com/mseng/TechnicalContent/_workitems/edit/1657424

### CODE REVIEW

File: docs/framework/winforms/controls/how-to-anchor-and-dock-child-controls-in-a-flowlayoutpanel-control.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "How to: Anchor and Dock Child Controls in a FlowLayoutPanel Control"
+description: Learn how to programatically anchor and dock child controls in a Windows Forms FlowLayoutPanel control.
```
Comment (adegeo) [inline docs/framework/winforms/controls/how-to-anchor-and-dock-child-controls-in-a-flowlayoutpanel-control.md:3]: I'm seeing this spelling error in quite a few places.

```suggestion
description: Learn how to programmatically anchor and dock child controls in a Windows Forms FlowLayoutPanel control.
```

File: docs/framework/winforms/advanced/how-to-create-mdi-parent-forms.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "How to: Create MDI Parent Forms"
+description: Learn how to create MDI parent Windows Forms programatically and by using the Windows Forms Designer.
```
Comment (adegeo) [inline docs/framework/winforms/advanced/how-to-create-mdi-parent-forms.md:3]:
```suggestion
description: Learn how to create an MDI parent form programmatically and by using the Windows Forms Designer.
```

File: docs/framework/winforms/advanced/how-to-print-a-windows-form.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "How to: Print a Windows Form"
+description: Learn how to programatically print a copy of the current Windows Form by using the CopyFromScreen method.
```
Comment (adegeo) [inline docs/framework/winforms/advanced/how-to-print-a-windows-form.md:3]:
```suggestion
description: Learn how to programmatically print a copy of the current Windows Form by using the CopyFromScreen method.
```

File: docs/framework/winforms/controls/how-to-view-multiple-lines-in-the-windows-forms-textbox-control.md [hunk 4]
```diff
@@ -1,5 +1,6 @@
 ---
 title: View Multiple Lines in TextBox Control
+description: Learn how to view multiple lines in the Windows Forms TextBox control by setting the Multiline, WordWrap, and ScrollBars properties to appropriate values.
```
Comment (adegeo) [inline docs/framework/winforms/controls/how-to-view-multiple-lines-in-the-windows-forms-textbox-control.md:3]: This change feels a bit better.
```suggestion
description: Learn how to view multiple lines in the Windows Forms TextBox control by setting the Multiline, WordWrap, and ScrollBars properties.
```

**Review by adegeo — CHANGES_REQUESTED** [review]
Thanks! Some minor issues to take care of.

**Comment by damabe** [comment]
@adegeo Thanks for the review. I've made the changes. Please sign off or merge. I don't have sign off permissions on this repo.

**Comment by damabe** [comment]
@adegeo Gentle reminder to review the updates and sign off when ready.

**Review by adegeo — APPROVED** [review]

**Comment by adegeo** [comment]
👍 Sorry about that.

**Pull request merged**